front/app/components/ConsentManager/destinations.ts

```diff
@@ -9,17 +12,13 @@ export type TCategory = (typeof CATEGORIES)[number];
 
 const destinationConfigs: IDestinationConfig[] = [];
 
-export interface IConsentManagerFeatureMap {}
-export type TConsentManagerFeature =
-  IConsentManagerFeatureMap[keyof IConsentManagerFeatureMap];
-
 export interface IDestinationConfig {
   /** A unique key, used to name the destination in the stored cookie */
   key: IDestination;
   /** Destinations are grouped in categories. Under which category should it be listed? */
   category: TCategory | ((tenant: IAppConfigurationData) => TCategory);
   /** The name of the feature flag that should be active for the destination to be functional */
-  feature_flag?: TConsentManagerFeature;
+  feature_flag?: TAppConfigurationSetting;
```

Comment:
Now we get a warning whenever we'd remove a "consent manager destination" from app config.